velox/connectors/hive/CMakeLists.txt

```diff
@@ -16,3 +16,8 @@ add_library(velox_hive_connector OBJECT HiveConnector.cpp FileHandle.cpp)
 
 target_link_libraries(velox_hive_connector velox_connector
                       velox_dwio_dwrf_reader velox_dwio_dwrf_writer file)
+
+add_executable(filehandle_test ./tests/FileHandleTest.cpp)
```

Comment:
Let's move this to velox/connectors/hive/tests/CMakeLists.txt and add velox_ prefix to target name or add the file to the existing velox_hive_connector_test

Reply:
Hi Masha, can you point me to the existing velox_hive_connector_test? Thanks.

Reply:
Found it. I missed the latest commits.